src/databricks/labs/ucx/workspace_access/scim.py

```diff
@@ -94,22 +95,31 @@ def _crawler_task(group: iam.Group, property_name: str):
             raw=json.dumps([e.as_dict() for e in getattr(group, property_name)]),
         )
 
-    def _inflight_check(self, group_id: str, value: list[iam.ComplexValue], property_name: str):
+    def verify(
```

Comment:
Okay, I see what you're mean by incompatibility. So, in this case: keep in_flight_check() methods mostly as is and call them from verify() method, which has to have a very narrow interface.